integration/helper_test.go

```diff
@@ -433,6 +476,65 @@ options:
 			Expect(err).To(HaveOccurred())
 			assertErrorsHandled()
 		})
+		It("skips batches if skip file is discovered with single datafile config", func() {
+			args := []string{
+				"--toc-file", tocFile,
+				"--pipe-file", pipeFile,
+				"--content", "1",
+				"--single-data-file",
+				"--restore-agent",
+				"--oid-file", restoreOidFile,
+				"--data-file", dataFileFullPath + ".gz",
+				"--on-error-continue",
+			}
+			doTestSkipFiles(-1, false, args)
```

Comment:
why not use `args...` like `exec.Command`?

Reply:
I believe there is no need to more sophisticated language construct here.

Reply:
only change
```go
doTestSkipFiles(-1, false, args...)
```
and
```go
func doTestSkipFiles(oid int, withPlugin bool, args ...string) {
```

Reply:
why is it better and important?

Reply:
It works fine with an array, and array is easy to manipulate on. I still prefer an old-fashioned unix-style array.

Reply:
with `args...` we may do following:
```go
func doTestSkipFiles(oid int, withPlugin bool, args ...string) {
	append([]string{
		"--toc-file", tocFile,
		"--pipe-file", pipeFile,
		"--content", "1",
		"--restore-agent",
		"--oid-file", restoreOidFile,
		"--data-file", dataFileFullPath + ".gz",
		"--on-error-continue",
	}, args...)
...
```
and in tests
```go
doTestSkipFiles(-1, false, "--single-data-file")
...
doTestSkipFiles(-1, false, "--resize-cluster", "--orig-seg-count", "6", "--dest-seg-count", "3")
```